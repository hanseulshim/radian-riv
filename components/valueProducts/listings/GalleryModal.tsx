import React from 'react'
import Modal from 'components/common/Modal'
interface Props {
  closeModal: () => void
  photos: string[]
}

export default function GalleryModal({ closeModal, photos }: Props) {
  return (
    <Modal closeModal={closeModal} title={''} percent={90} id="gallery-modal">
      <div className="gallery-container">
        {photos.map(photo => (
          <div key={photo} className="gallery-wrapper">
            <img className="photo" src={photo} alt="property" />
          </div>
        ))}
      </div>
    </Modal>
  )
}
