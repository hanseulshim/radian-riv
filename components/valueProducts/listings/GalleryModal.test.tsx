import * as React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import GalleryModal from './GalleryModal'

describe('GalleryModal', () => {
  let component
  beforeAll(() => {
    const props = {
      closeModal: () => {},
      photos: []
    }
    component = shallow(<GalleryModal {...props} />)
  })
  it('should match snapshot', () => {
    expect(toJson(component)).toMatchSnapshot()
  })
})
