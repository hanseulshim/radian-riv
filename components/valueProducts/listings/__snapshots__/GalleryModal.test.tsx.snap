// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GalleryModal should match snapshot 1`] = `
<Modal
  closeModal={[Function]}
  id="gallery-modal"
  percent={90}
  title=""
>
  <div
    className="gallery-container"
  />
</Modal>
`;
